assets/js/hooks/index.ts

```diff
@@ -18,6 +18,19 @@ export {
   TabbedPanels,
 };
 
+export const Toggle = {
+  mounted() {
+    const checkbox = this.el.querySelector('input[type="checkbox"]');
+
+    if (checkbox?.disabled) return;
+
+    this.el.addEventListener('click', () => {
+      checkbox.checked = !checkbox.checked;
+      checkbox.dispatchEvent(new Event('change', { bubbles: true }));
+    });
+  },
+};
+
```

Comment:
Do we still need this? As far as I can tell the checkbox element gets updated without this event listener - since there is a call back to the LiveView. Or is this for situations (we aren't using this elsewhere like this), where the toggle is part of a form and we aren't calling back to the LiveView for the new checked state?

Reply:
This was the initial approach but I just forgot to remove it. Apologies and thanks for the catch. The toggle works very well without it.